build.xml

```diff
@@ -66,18 +36,18 @@
     </target>
 
     <target name="compile-cubrid" depends="src-cubrid">
-        <javac destdir="${bin-cubrid}" source="1.5" target="1.5" encoding="EUC-KR" debug="true" debuglevel="lines,source,vars" deprecation="off" includeantruntime="no">
+        <javac destdir="${bin-cubrid}" source="1.6" target="1.6" encoding="EUC-KR" debug="true" debuglevel="lines,source,vars" deprecation="off" includeantruntime="no">
             <src path="${src-cubrid}"/>
             <include name="**/*.java"/>
         </javac>
     </target>
```

Comment:
Here, the java files are already set in `${src-cubrid}`. By `<include name="**/*.java"/>`, it may include the java files which un-filtered and compile them.